src/style.css

```diff
@@ -0,0 +1,33 @@
+.important {
+  background-color: red;
+}
+
+.interesting {
+  background-color: green;
+}
+
+.surprising {
+  background-color: orange;
+}
+
+#header {
+  background-color: blue;
+}
+
+#footer {
+  background-color: black;
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+
+div {
+  margin-left: auto;
+  margin-right: auto;
+  width: 300px;
+  height: 100px;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
first of all, body should be on the top of the document; and secondly padding is already 0